src/util/arr/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Build the `arr` module as a standalone static library
+# This is a temporary requirement to allow us to benchmark the
+# Rust implementation of the triemap against the original C implementation.
+file(GLOB ARR_SOURCES "arr.c")
+add_library(arr STATIC ${ARR_SOURCES})
+target_include_directories(arr PRIVATE . ../..)
```

Comment:
Why is this needed ?

Reply:
We need to invoke `array_new_sz` [here](https://github.com/RediSearch/RediSearch/pull/6119/files#diff-a01d2d8e17d9a560a800037f2d3d372a69cf4e11d6ec3292e0f4be69950b44beR53) and `array_free` [down here](https://github.com/RediSearch/RediSearch/pull/6119/files#diff-a01d2d8e17d9a560a800037f2d3d372a69cf4e11d6ec3292e0f4be69950b44beR86).
Since we're building `deps/triemap.c` as a static lib, it will refer to the symbols in `arr`, but it won't include their definition. We need to build the `arr` module as its own static lib to get those definitions in.